server/infrastructure/auth_repository.py

```
@@ -0,0 +1,132 @@
+        except Exception as e:
+            raise HTTPException(
+                status_code=status.HTTP_401_UNAUTHORIZED,
+                detail=f"Invalid or expired token: {str(e)}"
+            )
+    
+    async def store_user_data(self, user_id: str, user_data: Dict) -> bool:
+        """Store additional user data"""
+        try:
+            user_ref = self.db.collection('users').document(user_id)
+            user_ref.set(user_data)
+            return True
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)